native_client/ctcdecode/ctc_beam_search_decoder.cpp

```diff
@@ -170,7 +170,8 @@ DecoderState::decode() const
     for (size_t i = 0; i < beam_size_ && i < prefixes_copy.size(); ++i) {
       auto prefix = prefixes_copy[i];
       if (prefix->is_empty()) {
-        scores[prefix] = OOV_SCORE;
+        // TODO: doing 'scores[prefix] = OOV_SCORE;'
+        // generates 'i' instead of silences
```

Comment:
No need to leave the TODO here, we can just outright remove the conditional.

Reply:
So this does not regresses any UTF-8 work you did, right ?

Reply:
No, and even if it does I'd have to find a proper solution anyway.